.github/workflows/codeowners-validation.yml

```diff
@@ -0,0 +1,56 @@
+name: CODEOWNERS Validation
+run-name: ${{ github.actor }} is checking each Dockerfile for a CODEOWNER
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  CheckDockerfileCodeowners:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - name: Check each Dockerfile for a CODEOWNER
+        run: |
+          #!/bin/bash
```

Comment:
Consider putting this script in its own file so that it can be run locally. Additionally, using PowerShell instead of bash would be more accessible to more .NET devs, since running bash on Windows takes some extra steps.

Reply:
PowerShell has it's downfalls IMO as well (e.g. is it available on the build agent?).  While I agree pulling this into a script is a good decision, the benefit is testability not a general dev workflow therefore I don't see cross platform as being necessary.

Reply:
Bash is fine with me as well, but it's worth noting that GitHub hosted runners use the same set of software as Microsoft-hosted AzDO runners, which includes PowerShell: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#powershell-tools

Reply:
I went with using bash given the general consensus in this thread. Let me know if I should change to PowerShell. For now, I'm resolving this thread.